wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -207,16 +207,14 @@ class WirePlugin : Plugin<Project> {
         task.description = "Generate protobuf implementation for ${source.name}"
 
         // Flatten all the input files here. Changes to any of them will cause the task to re-run.
-        task.source(
-          buildList {
-            for (rootSet in protoSourceProtoRootSets) {
-              addAll(rootSet.roots)
-            }
-            for (rootSet in protoPathProtoRootSets) {
-              addAll(rootSet.roots)
-            }
-          },
-        )
+        for (rootSet in protoSourceProtoRootSets) {
```

Comment:
Calling `.roots` resolves the configuration; but passing a configuration is lazy!